scripts/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This config file is for the node scripts only, we need different config for those than for our components in `src`